telefunc/tsconfig.tests.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.base.json",
   "include": ["./**/*.spec.ts"],
+  "files": ["node/server/shield/codegen/types.ts"],
```

Comment:
Any reason we don't add it at `telefunc/node/tsconfig.json` instead?

Reply:
It's already included there.

> Btw. a potential solution for the problem of the emit info being truncated:
> 
> I can see that the threshold for truncating to be some `const` variable in TypeScript.
> 
> If that's the case then it's easy to change it to some big number `9999` in a post-install script. (I believe TypeScript ships as JavaScript and not binary; should be easy to patch TypeScript.)
> 
> We pin TypeScript and check if the patch still works when we use a newer TypeScript version.

I'm using `.getLiteralValue()` in a way that seems to work consistently now!

Reply:
That `tsconfig.json` does not apply to the tests.

Reply:
Before running the tests all TS files are transpiled. So I believe it should work to add it at `telefunc/node/tsconfig.json` instead. It would be cleaner (`types.ts` is an integrated part of the `node/` build; it's not only used for tests).

Reply:
Yes but they are not transpiles to the `tests` folder. Right now `types.d.ts` can be imported with `${__dirname}/types.d.ts` and something can be said for that as well. Feel free to change it though.

Reply:
You're right, my bad.